src/screens/PostPageDetail/index.js

```diff
@@ -10,8 +10,16 @@ import {setFeedByIndex} from '../../context/actions/feeds';
 
 const FeedsPostDetail = (props) => {
   const [feedsContext, dispatch] = React.useContext(Context).feeds;
-  const {feedId, refreshCache, haveSeeMore, refreshParent, contextSource, data, isKeyboardOpen} =
-    props.route.params;
+  const {
+    feedId,
+    refreshCache,
+    haveSeeMore,
+    refreshParent,
+    contextSource,
+    data,
+    isKeyboardOpen,
+    fromFeeds
+  } = props.route.params;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `fromFeeds` to the destructured props from `props.route.params` is a good way to pass down this new property. However, it's important to ensure that `fromFeeds` is always defined in the route params to avoid potential issues with `undefined` values. If there's a chance that `fromFeeds` might not be provided, consider providing a default value during destructuring.
```diff
-    fromFeeds
+    fromFeeds = false
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->